editor/debugger/editor_visual_profiler.cpp

```diff
@@ -143,25 +143,26 @@ void EditorVisualProfiler::_item_selected() {
 }
 
 void EditorVisualProfiler::_update_plot() {
-	int w = graph->get_size().width;
-	int h = graph->get_size().height;
+	const int w = graph->get_size().width;
+	const int h = graph->get_size().height;
 
 	bool reset_texture = false;
 
-	int desired_len = w * h * 4;
+	const int desired_len = w * h * 4;
 
 	if (graph_image.size() != desired_len) {
 		reset_texture = true;
 		graph_image.resize(desired_len);
 	}
 
 	uint8_t *wr = graph_image.ptrw();
+	const Color background_color = get_theme_color("dark_color_2", "Editor");
 
-	//clear
+	// Clear the previous frame and set the background color.
 	for (int i = 0; i < desired_len; i += 4) {
-		wr[i + 0] = 0;
-		wr[i + 1] = 0;
-		wr[i + 2] = 0;
+		wr[i + 0] = Math::fast_ftoi(background_color.r * 255);
```

Comment:
Is `fast_ftoi` something that we explicitly use in other similar places? Is this critical enough to warrant using this instead of a simple cast?

Reply:
It's also used in the general profiler. I guess it's used because this is used for updating an image every frame, so performance is paramount.